class_room.rb

```diff
@@ -0,0 +1,13 @@
+class ClassRoom
+  attr_accessor :label, :students
```

Comment:
- [ ] Kindly define the students as `attr_reader` instead of `attr_accessor` as you don't need to modify it outside the class.